packages/payload/src/uploads/types.ts

```diff
@@ -12,6 +12,14 @@ export type FileSize = {
   width: null | number
 }
 
+/**
+ * FileSize_P4 is a more precise type, and will replace FileSize in Payload v4.
```

Comment:
What is this naming convention? Feels like `_P4` doesn't feel meaningful. What is the intent?

Reply:
As the comment says, this is to indicate that it will be the type of v4 of Payload. I originally used `_V4`, but changed it to `_P4` so that no one thinks it is the version of this type, but the version of Payload.
Alternatively I could use `_Payload_V4` or `_Payload_v4`. Any preferences?